intelmq/bots/collectors/ftp/collector_ftps.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""
+FTPS collector bot
+
+Parameters:
+ftps_host: string
+ftps_port: number
+ftps_username, ftps_password: string
+ftps_directory: string
+ftps_file: string
+
+
+"""
+from __future__ import unicode_literals
+import sys
+from ftplib import FTP_TLS
+import socket
+import ssl
+import zipfile
+import io
+
+from intelmq.lib.bot import Bot
+from intelmq.lib.harmonization import DateTime
+from intelmq.lib.message import Report
+
+
+# https://stackoverflow.com/questions/12164470/python-ftp-implicit-tls-connection-issue
```

Comment:
If you copied code from there, attribution is required under cc by-sa 3.0: https://blog.stackoverflow.com/2009/06/attribution-required/